steampy/utils.py

```diff
@@ -59,6 +62,15 @@ def merge_items_with_descriptions_from_offer(offer: dict, descriptions: dict) ->
     return offer
 
 
+def merge_items_with_descriptions_from_listing(listings: dict, ids_to_assets_address: dict,
+                                               descriptions: dict) -> dict:
+    for listing_id, listing in listings.get("sell_listings").items():
+        asset_address = ids_to_assets_address[listing_id]
+        description = descriptions[asset_address[0]][asset_address[1]][asset_address[2]]
+        listing["description"] = description
+    return listings
+
```

Comment:
Only the sell listings have the asset information, so first we get an "asset_address" and with this we can retrieve the correct description and add it to the listing.